config/v1/0000_10_config-operator_01_oauth.crd.yaml

```diff
@@ -642,9 +642,10 @@ spec:
                     the maximum amount of time that can occur between consecutive
                     uses of the token. Tokens become invalid if they are not used
                     within this temporal window. The user will need to acquire a new
-                    token to regain access once a token times out. If this value is
-                    not set, then tokens are valid until their expiry. Takes valid
-                    time duration string such as "5m", "1.5h" or "2h45m".
+                    token to regain access once a token times out. Takes valid time
+                    duration string such as "5m", "1.5h" or "2h45m". The current minimum
+                    allowed value for duration is 300s (5 minutes). Tokens never timeout
+                    otherwise unless configured per oauth client.
```

Comment:
Would be clear about zero. "If not specified or zero and the client does not override the value, token never time out."

Reply:
Question is whether we want to allow zero at all. Actually we don't have to if it is a pointer with omitempty anyway.

Reply:
we don't want to allow zero. to be more specific any value less than minimum activity timeout. otherwise it becomes complicated again if we allow zero but does not allow any value in the range (0, 300) and allow values greater than 300s. It can be nil if user does not want token to timeout.

Reply:
agree.